src/go/k8s/pkg/resources/headless_service.go

```diff
@@ -46,12 +47,14 @@ func NewHeadlessService(
 	client k8sclient.Client,
 	pandaCluster *redpandav1alpha1.Cluster,
 	scheme *runtime.Scheme,
+	svcPorts map[string]int,
```

Comment:
Why headless port have admin port? Isn't that unnecessary?

Reply:
I think it is, let's just keep it there though